### DISCUSSION TITLE
No IntelliJ JSP support

### DISCUSSION THREAD

**Discussion by Swofty-Developments** [community/community#7035] [Copilot Conversations]
Hey. So I was messing around with Copilot and everything was going fine, I then load into my JSP file, and even though it'll give me suggestions while coding, it won't actually take comments and give me feedback - this is true for both the HTML and JS. I'm wondering if there is any way to force it to be on for JS in my JSP file.

**Comment by jansorg** [comment]
Thanks for reporting! Could you provide a snippet of JSP, where you'd expect JS completions?
Are you using language injections are just plain attributes or tags?

**Reply by Swofty-Developments** [reply]
![image](https://user-images.githubusercontent.com/75875764/139532630-5eb9abff-3a2e-49de-a2ec-9809de9bb967.png)
Hey, thanks for your response. I'm using plain tags.
